### PR TITLE
Issue 65 - Add Twitter Cards metadata

### LINKED ISSUE #65: Twitter Cards metadata is not being generated
After using Twitter's [Card Validator](https://cards-dev.twitter.com/validator) I discovered that metadata is not being generated automatically.

This is what I had to add for a custom content type on a customer's site:

``` xml
  <head>
    <metal:head fill-slot="head_slot">
      <meta name="twitter:card" content="summary_large_image">
      <meta name="twitter:site" content="@customer">
      <meta name="twitter:title" content="Title"
          tal:attributes="content context/Title">
      <meta name="twitter:description" content="Description"
          tal:attributes="content context/Description">
    </metal:head>
  </head>
```

we have to think about Plone 5 implementation to make migration easier.

### PR DESCRIPTION
**Pull request by rodfersou**
closes #65

### CODE REVIEW

**Comment by rodfersou** [comment]
There are more markup we can fill here https://dev.twitter.com/cards/markup

Maybe the ``twitter:creator`` is a good idea.

File: sc/social/like/plugins/twitter/browser.py [hunk 1]
```diff
@@ -35,6 +38,7 @@ def __init__(self, context, request):
         self.url = context.absolute_url()
         self.language = get_language(context)
         self.sheet = getattr(pp, 'sc_social_likes_properties', None)
+        self.image = get_content_image(context, width=1200, height=630)
```
Comment (hvelarde) [inline sc/social/like/plugins/twitter/browser.py]: @rodfersou why are you hardcoding this with those valued?

File: sc/social/like/plugins/twitter/browser.py [hunk 2]
```diff
@@ -57,3 +61,12 @@ def share_link(self):
...
         return url
+
+    def image_url(self):
+        """ Return url to image
+        """
+        img = self.image
+        if img:
+            return img.url
+        else:
+            return '%s/logo.png' % self.site_url
```
Comment (hvelarde) [inline sc/social/like/plugins/twitter/browser.py]: @rodfersou why are you hardcoding a link to a `logo.png` image?

File: sc/social/like/plugins/twitter/templates/metadata.pt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<tal:twitter>
+  <meta name="twitter:card" content="summary_large_image" />
+  <meta name="twitter:image" tal:attributes="content view/image_url" />
+  <meta name="twitter:image:alt" tal:attributes="content view/title|view/description" />
```
Comment (hvelarde) [inline sc/social/like/plugins/twitter/templates/metadata.pt]: @rodfersou this is not needed

File: sc/social/like/plugins/twitter/templates/metadata.pt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<tal:twitter>
+  <meta name="twitter:card" content="summary_large_image" />
+  <meta name="twitter:image" tal:attributes="content view/image_url" />
```
Comment (hvelarde) [inline sc/social/like/plugins/twitter/templates/metadata.pt]: @rodfersou this should be conditional

**Pull request merged**